srcpkgs/python3-PGPy/template

```diff
@@ -1,18 +1,17 @@
 # Template file for 'python3-PGPy'
 pkgname=python3-PGPy
-version=0.5.4
-revision=2
+version=0.6.0
+revision=1
 build_style=python3-module
 hostmakedepends="python3-setuptools python3-wheel"
 depends="python3-cryptography python3-pyasn1 python3-six"
+checkdepends="tox python3-cryptography python3-six python3-pyasn1 python3-progressbar"
```

Comment:
```suggestion
checkdepends="python3-pytest python3-progressbar $depends"
```
Tox is no good because it fetches packages in a virtualenv rather than using packages that we've built for Void.